src/sentry/logging/audit.py

```diff
@@ -35,14 +38,23 @@ def log_entry(entry):
     """
     Give an AuditLogEntry object to the audit logger.
     """
-    log(
-        organization_id=entry.organization_id,
-        actor_id=entry.actor_id,
-        actor_key=entry.actor_key,
-        target_object=entry.target_object,
-        target_user_id=entry.target_user_id,
-        event=entry.get_event_display(),
-        ip_address=entry.ip_address,
-        data=entry.data,
-        datetime=entry.datetime,
-    )
+    if fmt is 'human':
+        logger.info(
+            "[Audit Log] [{org}] {user} {note}".format(
+                org=entry.organization.name,
```

Comment:
I'm still not a fan of doing this since this _potentially_ creates a query just to get the name. Then below as well, so it's possible to be introducing two queries, just for a log entry.